core/src/main/resources/jenkins/security/ApiTokenProperty/help-tokenStore.html

```diff
@@ -0,0 +1,12 @@
+<div>
+    The API tokens can be used for authenticating yourself in the REST API call.
+    See <a href="https://jenkins.io/redirect/api-token">our wiki</a> for more details.<br/>
+    The associated username with the tokens is the regular one of your account.<br/>
```

Comment:
I would rephrase this to: The username associated with each token is your Jenkins username.